content/en/blog/2019/istioctl-describe/index.md

```diff
@@ -0,0 +1,244 @@
+---
+title: "'istioctl describe' in Istio 1.3"
+description: Introducing a command-line utility to find configuration affecting pod traffic.
+publishdate: 2019-09-20
+attribution: Ed Snible (IBM)
+keywords: [traffic-management, istioctl, debugging]
+---
+
+Istio 1.3 includes an experimental command-line tool designed to help find and
+understand configuration that affects traffic to a pod.
+
+`istioctl experimental describe pod <pod>` shows the Istio configuration that affects a pod.
+
+The team has prepared a mini-tutorial to demonstrate the features and use of the tool.
+
+## describe tutorial using Bookinfo tutorial
+
+First, let us [deploy Bookinfo](/docs/examples/bookinfo/).
+
+{{< text bash >}}
+$ kubectl apply -f samples/bookinfo/platform/kube/bookinfo.yaml
+$ kubectl apply -f samples/bookinfo/networking/bookinfo-gateway.yaml
+$ export INGRESS_HOST=$(kubectl -n istio-system get service istio-ingressgateway -o jsonpath='{.status.loadBalancer.ingress[0].ip}')
+$ export INGRESS_PORT=$(kubectl -n istio-system get service istio-ingressgateway -o jsonpath='{.spec.ports[?(@.name=="http2")].port}')
+$ export GATEWAY_URL=$INGRESS_HOST:$INGRESS_PORT
+$ echo Bookinfo at $GATEWAY_URL/productpage
+{{< /text >}}
+
+Let's describe a pod:
+
+{{< text bash >}}
+$ export RATINGS_POD=$(kubectl get pod -l app=ratings -o jsonpath='{.items[0].metadata.name}')
+$ istioctl experimental describe pod $RATINGS_POD
+{{< /text >}}
+
+The output tells us which containers the pod exposes, the Istio protocol for the microservice on port 9080, and the mutual TLS settings for the pod.
+
+{{< text plain >}}
+Pod: ratings-v1-f745cf57b-qrxl2
+   Pod Ports: 9080 (ratings), 15090 (istio-proxy)
+--------------------
+Service: ratings
+   Port: http 9080/HTTP
+Pilot reports that pod enforces HTTP/mTLS and clients speak HTTP
+{{< /text >}}
+
+## `DestinationRules`
```

Comment:
Do not use the value of the `kind:` field unless you are referencing the value directly. See our style guide on [consistent capitalization](https://istio.io/about/contribute/style-guide/#use-consistent-capitalization)